mappings/net/minecraft/client/network/ClientPlayerEntity.mapping

```diff
@@ -9,9 +9,18 @@ CLASS djf net/minecraft/client/network/ClientPlayerEntity
 	FIELD cd recipeBook Lcul;
 	FIELD ce tickables Ljava/util/List;
 	FIELD cf clientPermissionLevel I
+	FIELD cg lastX D
+	FIELD ch lastBaseY D
+	FIELD ci lastZ D
+	FIELD cj lastYaw F
+	FIELD ck lastPitch F
+	FIELD cl lastOnGround Z
+	FIELD cm lastIsHoldingSneakKey Z
```

Comment:
Could this be `lastSneaking`, to match `lastSprinting`, or is it treated differently?

Reply:
We already have lastSneaking, which is whether the player was sneaking last tick. This is whether the sneak key was down last tick (the player may be sneaking without holding sneak if it's in a <2 block tall space).